web/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM openmicroscopy/omero-ssh-systemd
+
+MAINTAINER OME
+
+ENV JENKINS_SWARM_VERSION 2.0
+ENV LANG en_US.UTF-8
+ARG JAVAVER=openjdk18
+
+
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
+# Download and run omero-install. 
+ENV OMERO_INSTALL /tmp/omero-install/linux
+
+RUN yum install -y git && \
+    git clone -b java-parameters https://github.com/jburel/omero-install.git /tmp/omero-install
```

Comment:
@sbesson had a good idea: down the road, we should likely turn this into a build-arg, but I think that will require a newer version of docker-compose.